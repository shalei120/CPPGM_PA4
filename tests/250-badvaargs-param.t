#define A(__VA_ARGS__) X
//A(1)
