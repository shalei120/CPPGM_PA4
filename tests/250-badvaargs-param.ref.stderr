ERROR: __VA_ARGS__ in macro parameter list
